app/views/index.html

```diff
@@ -16,8 +16,7 @@ <h1 class="heading-xlarge">
       </h1>
 
       <ul>
-        <li><a href="/example">Example Link</a></li>
-        <li><a href="/my-utilisation">My Utilisation</a></li>
+        <li><a href="/offender-manager-utilisation/1/2017">My Utilisation</a></li>
```

Comment:
I think this should be called `caseload-utilisation` as although the current scope is to filter for an individual Offender Manager it will be expanded to include filters for Team, LDU etc.

See https://noms-wmt-ux.herokuapp.com/alpha/v3-2/capacity

This also applies to the 'past'/'current' description in the current taiga stories I think - it's referring to the filters on the same view effectively.